go.mod

```diff
@@ -36,8 +35,8 @@ require (
 
 replace (
 	// Pin RHsyseng library versions
-	github.com/RHsyseng/console-cr-form => github.com/RHsyseng/console-cr-form v0.0.0-20200414161125-135bc9b52976
-	github.com/RHsyseng/operator-utils => github.com/RHsyseng/operator-utils v0.0.0-20200407154515-94a3ad1cdcfc
+	github.com/RHsyseng/console-cr-form => github.com/RHsyseng/console-cr-form v0.0.0-20200129200812-dc119cb0bd4d
```

Comment:
why is this being reverted to an older version? is this module related to this PR somehow?